src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -875,6 +871,27 @@ describe("Routes/ConfirmBid", () => {
         user_id: "my-user-id",
       })
     })
+
+    it("displays an error when there are validation errors in the form", async () => {
+      const env = setupTestEnv()
+      const page = await env.buildPage({
+        mockData: FixtureForUnregisteredUserWithoutCreditCard,
+      })
+
+      createTokenMock.mockResolvedValue(stripeTokenResponse)
+      env.mutations.useResultsOnce(createCreditCardAndUpdatePhoneSuccessful)
+      env.mutations.useResultsOnce(createBidderPositionSuccessfulAndBidder)
+      mockBidderPositionQuery.mockResolvedValue(
+        confirmBidBidderPositionQueryWithOutbid
+      )
+
+      await page.fillFormWithValidValues()
+      await page.agreeToTerms()
+      await page.submitForm()
+
+      expect(window.location.assign).not.toHaveBeenCalled()
+      expect(page.text()).toContain("Your bid wasn’t high enough.")
+    })
```

Comment:
Sadly this test passes without the fix because of the difference between the actual impl and [the test setup](https://github.com/artsy/reaction/blob/ed7f6c3fd52102becb8bec90968add28fc5766c9/src/DevTools/createTestEnv.tsx#L222).

Reply:
Can you elaborate a bit? Is there a way we can get the test to fail legitimately?